openmetadata-ui/src/main/resources/ui/src/pages/AlertDetailsPage/AlertDetailsPage.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ *  Copyright 2022 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { Card } from 'antd';
+import { trim } from 'lodash';
+import React, { useEffect, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import {
+  getAlertActionForAlerts,
+  getAlertsFromId,
+} from '../../axiosAPIs/alertsAPI';
+import { AlertDetailsComponent } from '../../components/Alerts/AlertsDetails/AlertDetails.component';
+import DeleteWidgetModal from '../../components/common/DeleteWidget/DeleteWidgetModal';
+import {
+  GlobalSettingOptions,
+  GlobalSettingsMenuCategory,
+} from '../../constants/GlobalSettings.constants';
+import { EntityType } from '../../enums/entity.enum';
+import { AlertAction } from '../../generated/alerts/alertAction';
+import { AlertFilterRule, Alerts } from '../../generated/alerts/alerts';
+import { getEntityName } from '../../utils/CommonUtils';
+import { getSettingPath } from '../../utils/RouterUtils';
+import { showErrorToast } from '../../utils/ToastUtils';
+
+const AlertDetailsPage = () => {
+  const { t } = useTranslation();
+
+  const { fqn } = useParams<{ fqn: string }>();
+  const [loadingCount, setLoadingCount] = useState(0);
+  const [alerts, setAlerts] = useState<Alerts>();
+  const [alertActions, setAlertActions] = useState<AlertAction[]>([]);
+  const [showDeleteModel, setShowDeleteModel] = useState(false);
+
+  const fetchAlert = async () => {
+    try {
+      setLoadingCount((count) => count + 1);
+
+      const response: Alerts = await getAlertsFromId(fqn);
+      const alertActions = await getAlertActionForAlerts(response.id);
+
+      const requestFilteringRules =
+        response.filteringRules?.map((curr) => {
+          const [fullyQualifiedName, filterRule] = curr.condition.split('(');
```

Comment:
Can we guard this for if `condition` is not present?

Reply:
It will be there

Reply:
But let me add it